test/extended/util/annotate/rules.go

```diff
@@ -209,6 +209,7 @@ var (
 			`\[sig-network\] Networking Granular Checks: Services should function for endpoint-Service`,
 			// https://github.com/ovn-org/ovn-kubernetes/issues/928
 			`\[sig-network\] Services should be rejected when no endpoints exist`,
+			`\[cli\] Oc explain networking types should contain proper fields description for special networking types`,
```

Comment:
Still not skipped in OVN test. Perhaps due to a typo: `oc` needs to be lowercase?

Reply:
I actually was following the perceived formatting rules and thought that it needs to be `Oc` instead of `oc`. @smarterclayton wdyt?

Reply:
@abhat IIRC they do need to match case; it's just a regex against the test name. So the exclusion explicitly handles case, it needs to match the test Describe.

Reply:
pushed a fix to match the case.